machine-learning/03-Training and AutoML.ipynb

```diff
@@ -1,45 +1,561 @@
+          "languageId": "dotnet-interactive.csharp"
+        }
+      },
+      "outputs": [
+        {
+          "data": {
+            "text/html": [
+              "<div><div><strong>Restore sources</strong><ul><li><span>https://mlnetcli.blob.core.windows.net/mlnetcli/index.json</span></li><li><span>https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json</span></li><li><span>https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json</span></li><li><span>https://pkgs.dev.azure.com/dnceng/public/_packaging/MachineLearning/nuget/v3/index.json</span></li></ul></div><div></div><div><strong>Installed Packages</strong><ul><li><span>Microsoft.Data.Analysis, 0.20.0-preview.22226.2</span></li><li><span>Microsoft.ML.AutoML, 0.20.0-preview.22226.2</span></li><li><span>MLNetAutoML.InteractiveExtension, 0.1.1</span></li><li><span>XPlot.Plotly.Interactive, 4.0.6</span></li></ul></div></div>"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {
+            "text/markdown": [
+              "Loading extensions from `MLNetAutoML.InteractiveExtension.dll`"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {
+            "text/markdown": [
+              "Loading extensions from `XPlot.Plotly.Interactive.dll`"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {
+            "text/markdown": [
+              "Configuring PowerShell Kernel for XPlot.Plotly integration."
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {
+            "text/markdown": [
+              "Installed support for XPlot.Plotly."
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        },
+        {
+          "data": {
+            "text/markdown": [
+              "Loading extensions from `Microsoft.Data.Analysis.Interactive.dll`"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        }
+      ],
+      "source": [
+        "// install dependencies and import using statement\n",
+        "#i \"nuget:https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json\" \n",
+        "#i \"nuget:https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json\"\n",
+        "#i \"nuget:https://mlnetcli.blob.core.windows.net/mlnetcli/index.json\"\n",
+        "\n",
+        "// add nightly build for ml.net\n",
+        "#i \"nuget:https://pkgs.dev.azure.com/dnceng/public/_packaging/MachineLearning/nuget/v3/index.json\"\n",
+        "\n",
+        "#r \"nuget:MLNetAutoML.InteractiveExtension,0.1.1\"\n",
+        "#r \"nuget:XPlot.Plotly.Interactive,4.0.6\"\n",
```

Comment:
I would recommend splitting the #r sections and the `using` statements into separate cells. 

Also, let's try and use Plotly.NET as much as possible.

Reply:
i can add a comment to suggest using `Ploty.Net` for all plotting cases

Reply:
That's a good workaround for now. I'll raise an issue in Plotly.NET to track the request for improvements to C#.